src/interfaces/new-request-interface.ts

```diff
@@ -61,6 +60,12 @@ export interface NewRequestIF {
   request_action_cd: NrRequestActionCodes
   request_jurisdiction_cd: string
   requestExaminationOrProvideConsent: RequestOrConsentIF
+  search: {
+    business: BusinessSearchIF
+    companyType: CompanyTypes
+    jurisdiction: any
+    request: RequestActionsI
+  }
```

Comment:
I struggled for a while with reactivity on variables in the search mixin and the components sometimes not seeing the changes. Ultimately, it looks like there can only be 1 watcher or reactive statement for them. So, I moved the variables to the store instead, and now reactivity works as expected: everyone can see all changes,